modules/apps/object/object-test/src/testFunctional/paths/ObjectAction.path

```diff
@@ -52,6 +57,11 @@
 		<td>//span[contains(., 'Enable Condition')]/..//span[contains(@class, 'toggle-switch-check-bar')]//input[contains(@type, 'checkbox')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>AUTOCOMPLETE_INPUT</td>
+		<td>//div[@class='dropdown-menu autocomplete-dropdown-menu show']//button[@class='dropdown-item' and contains(., '${key_entryName}')]</td>
+		<td></td>
+	</tr>
```

Comment:
This feels a little fragile, especially if the class value changes. We can use a part of the value like `dropdown-menu` or even skip the `div` entirely and just use the button, as long as it doesn't find duplicate elements.

For now, it's fine but in the future please keep this in mind when designing paths. If you're not sure what I mean, schedule a meeting with me and I can explain further.